public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -368,6 +369,23 @@ export function GrafanaEvaluationBehaviorStep({
                   name="execErrState"
                 />
               </Field>
+              <Field
+                label={t('alerting.alert.missing-series-resolve', 'Missing series evaluations to resolve')}
+                description={t(
+                  'alerting.alert.description-missing-series-evaluations',
+                  'How many consecutive evaluation intervals with no data for a dimension must pass before the alert state is considered stale and automatically resolved'
```

Comment:
can we clarify somehow that if we use 0, we are setting to the [default value that is 2](https://github.com/grafana/grafana/pull/101184)?